packaging/build/Microsoft.NET.Build.Containers.targets

```diff
@@ -13,6 +13,23 @@
                           )">true</_IsSDKContainerAllowedVersion>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- Reference data about this project -->
+    <_AspNetCoreFrameworkReference>@(FrameworkReference->WithMetadataValue('Identity', 'Microsoft.AspNetCore.App')</_AspNetCoreFrameworkReference>
+    <_IsAspNet Condition=" '$(UsingMicrosoftNETSdkWeb)' == 'true' or '$(_AspNetCoreFrameworkReference)' != '' ">true</_IsAspNet>
```

Comment:
Unfortunately, I don't think this'll work. In MSBuild, properties are evaluated before items, so when this is evaluated there won't be any `@(FrameworkReference)` items to check for, and `$(_AspNetCoreFrameworkReference)` will be the literal string `@(FrameworkReference->WithMetadataValue('Identity', 'Microsoft.AspNetCore.App')` . . . which is not empty, so I think this will cause `_IsAspNet=true` always.

Reply:
I just spent a little while trying a few possible alternate configurations and just pushed up a new change that seems to be a decent middle ground. Rather than depending on root level properties, I've moved the base image properties into an additional dedicated target `ComputeContainerBaseImage` (and made `ComputeContainerConfig` depend on this new target) that explicitly returns the ContainerBaseImage. This makes the property slightly more complicated to retrieve from within VS, but still entirely possible.

I also streamlined the _IsAspNet property to just look for the FrameworkReference, rather than also checking for the property set by the Web SDK.

Reply:
I've tested with both AspNet Web SDK projects as well as console projects and am getting back the correct base image from the target (and it properly handles manually adding a FrameworkReference to 'Microsoft.AspnetCore.App'.